src/main/java/net/imagej/ops/descriptor3d/AbstractVertexInterpolator.java

```diff
@@ -0,0 +1,35 @@
+package net.imagej.ops.descriptor3d;
+
+
+import org.scijava.ItemIO;
+import org.scijava.plugin.Parameter;
+
+import net.imagej.ops.AbstractComputerOp;
+import net.imagej.ops.AbstractFunctionOp;
+import net.imagej.ops.AbstractOp;
+import net.imagej.ops.Op;
+
+/**
+ * This is the {@link AbstractVertexInterpolator}. A vertex interpolator
+ * computes the real coordinates based on the pixel intensities. 
+ * 
+ * @author Tim-Oliver Buchholz, University of Konstanz
+ *
+ */
+public abstract class AbstractVertexInterpolator extends AbstractOp implements VertexInterpolator {
```

Comment:
how does this relate to the bresenham implementation in imglib2-roi? maybe you can reuse this?